rust/datafusion/src/physical_plan/hash_aggregate.rs

```diff
@@ -374,6 +380,18 @@ impl RecordBatchReader for GroupedHashAggregateIterator {
                         col,
                         accums
                     ),
+                    DataType::LargeList(_) => {
+                        let len = array.len();
+                        let list_array =
+                            array.as_any().downcast_ref::<LargeListArray>().unwrap();
```

Comment:
nit: I'd prefer that we use `expect` rather than `unwrap` so we can add a meaningful message.

Reply:
Sounds good, will update.